src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalValue = days * 40;
```

Comment:
Consider using a constant for the daily rental cost instead of the hardcoded value '40'.